lib/matplotlib/stackplot.py

```diff
@@ -56,6 +56,13 @@ def stackplot(axes, x, *args,
     data : indexable object, optional
         DATA_PARAMETER_PLACEHOLDER
 
+    where: bool or array of bool, optional
+        Passed to `.Axes.fill_between` and defines where to exclude horizontal
+        regions from being filled. The filled regions are defined by the
+        coordinates `x[where]`. Can be either a single bool, an array of shape
+        (N,) or an array of shape (M, N).
+        Should be used together with `interpolate=True`.
```

Comment:
Can we check if `interpolate=True` is set and raise/warn or if the user does not pass this, correctly track interpolate?

Is this comment still relevant or is this no longer needed with the new simpler approach?